src/Goerli-DssSpell.sol

```diff
@@ -24,6 +24,10 @@ import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralOnboardingAction } from "./Goerli-DssSpellCollateralOnboarding.sol";
 
+import "dss-interfaces/dss/EndAbstract.sol";
+import "dss-interfaces/dss/IlkRegistryAbstract.sol";
+import "dss-interfaces/dss/FlashAbstract.sol";
```

Comment:
I thought we typically wrote out interfaces in main contracts like spells.  But if we are going to import should we do:

```suggestion
import { EndAbstract, IlkRegistryAbstract, FlashAbstract } from "dss-interfaces/dss/interfaces.sol";
```